daemon/algod/api/server/v2/utils.go

```diff
@@ -112,13 +100,9 @@ func nilToZero(numPtr *uint64) uint64 {
 
 func computeCreatableIndexInPayset(tx node.TxnWithStatus, txnCounter uint64, payset []transactions.SignedTxnWithAD) (cidx *uint64) {
 	// Compute transaction index in block
-	offset := -1
-	for idx, stxnib := range payset {
-		if tx.Txn.Txn.ID() == stxnib.Txn.ID() {
-			offset = idx
-			break
-		}
-	}
+	offset := slices.IndexFunc(payset, func(ad transactions.SignedTxnWithAD) bool {
+		return ad.Txn.ID() == tx.Txn.Txn.ID()
```

Comment:
I know you didn't write this, but oh gosh do we really compute `tx.Txn.Txn.ID()` each iteration of the loop? 😭 

Since you're already touching this code, can you lift this out of the loop?

Reply:
auggh my bad. I need to take it out.